docs/sources/alerting/notifications.md

```diff
@@ -75,20 +75,22 @@ able to access the image.
 
 {{< imgbox max-width="40%" img="/img/docs/v4/slack_notification.png" caption="Alerting Slack Notification" >}}
 
-To set up slack you need to configure an incoming webhook url at slack. You can follow their guide on how
+To set up Slack, you need to configure an incoming Slack webhook URL. You can follow their guide on how
 to do that [here](https://api.slack.com/incoming-webhooks). If you want to include screenshots of the firing alerts
```

Comment:
Please rewrite this so that it removes "here" as the link text. Per the Documentation Style Guide, we want to use descriptive linking text, such as the title of the page being linked to, rather than uninformative verbiage such as "here."

Reply:
@oddlittlebird The text in question isn't touched by my PR so I think it's out of scope for the review, but since it's such an uncontroversial change, sure.